### ISSUE TITLE
Lets find the new Maintainer and ruin his digital life (By exposing him)

### ISSUE THREAD

**Issue by thecrazy** [deanoemcke/thegreatsuspender#1321]
I say there has got to be some payback for trying to mess with us. The world has to know that **you don't mess with open source**.

**Comment by mpragliola** [comment]
@thecrazy is there any substantial proof that something shady was going on?

**Comment by nuggs** [comment]
> @thecrazy is there any substantial proof that something shady was going on?

https://github.com/greatsuspender/thegreatsuspender/issues/1263
https://www.reddit.com/r/programming/comments/ld6pck/psa_please_uninstall_the_great_suspender_chrome/

I'm not sure if ruining someone's life is the correct course of action but if they are committing advertising fraud, it would be good for reporting them to the proper authorities.

**Comment by mpragliola** [comment]
I totally agree, @nuggs . We don't need to resort to harassment or bullying, which I really despise as reactions. Still if there is something going on it's good to expose it and report it.

**Comment by LewisSpring** [comment]
I think this is the wrong way to look at this. There is no point ruining someone's life like that, no matter how serious this is. It could be doxxing or similar, and _also illegal_
We should be working out who is responsible for any acts _**that are illegal**_ and then report those to the relevant people.
_People should want to ruin their life professionally via the proper authorities, If you'd like to put it that way._

**Comment by tcvv** [comment]
At this point I would be grateful if it was known it was advertising fraud. From all the things this extension could be doing on our behalf or information it could be collecting or changing, that's the least of my concerns.

**Comment by iSWORD** [comment]
dig -t A owebanalytics.com
```
;; ANSWER SECTION:
owebanalytics.com.	85769	IN	A	142.4.10.16
```

> https://db-ip.com/142.4.10.16

![image](https://user-images.githubusercontent.com/1176779/107104743-a3726e00-682b-11eb-8ee3-730011993f6a.png)

If you lookup unifiedlayer.com you'll find a lot of articles on them hosting malicious content.

> https://threatintelligence.guardicore.com/domain/unifiedlayer.com
> https://ericasadun.com/2018/05/15/the-festering-realities-of-bluehost-in-which-i-learn-about-unifiedlayer-com/

They wouldn't have hosted their CDN there if they didn't have bad intentions.

They're definitely bad people and deserve to be punished.

**Comment by ivan1282ka** [comment]
Yeah! Lets find the new maintainer and punish it!  Wow how clever is that!
But how it happened that there is suddenly a new maintainer? 
I think there are plenty of guilt in the OLD maintainer! 
It is like the skype case. The original developers of skype just sold all of their users to Microsoft.
Here the case of TheGreatsuspender is the same! 

And there is one general question about entire "open source" movement. 
It seems there tho main purposes of the "OpenSource"  (Hippocraticaly hidden behind word "OPEN") . 
1. Developers to become a property (well payed) of the owner of the software they pretend to make alternative. 
2. if this fail, then to sold their users for money.  

And now these purposes are reached! 
And all the rest of us: 
Where are we going from there? 
Perhaps self made source? 

Where are we going from there?

**Comment by thecrazy** [comment]
> > @thecrazy is there any substantial proof that something shady was going on?
> 
> #1263
> https://www.reddit.com/r/programming/comments/ld6pck/psa_please_uninstall_the_great_suspender_chrome/
> 
> I'm not sure if ruining someone's life is the correct course of action but if they are committing advertising fraud, it would be good for reporting them to the proper authorities.

That's why I said "digital" I wouldn't actually condone ruining his "real" life. Guess I could have been clearer. But the guy should definitely feel the repercussions outside the confines of GitHub.

**Comment by thecrazy** [comment]
> dig -t A owebanalytics.com
> 
> ```
> ;; ANSWER SECTION:
> owebanalytics.com.	85769	IN	A	142.4.10.16
> ```
> 
> > https://db-ip.com/142.4.10.16
> 
> ![image](https://user-images.githubusercontent.com/1176779/107104743-a3726e00-682b-11eb-8ee3-730011993f6a.png)
> 
> If you lookup unifiedlayer.com you'll find a lot of articles on them hosting malicious content.
> 
> > https://threatintelligence.guardicore.com/domain/unifiedlayer.com
> > https://ericasadun.com/2018/05/15/the-festering-realities-of-bluehost-in-which-i-learn-about-unifiedlayer-com/
> 
> They wouldn't have hosted their CDN there if they didn't have bad intentions.
> 
> They're definitely bad people and deserve to be punished.

Thats a good find!

**Comment by TheThor** [comment]
The old maintainer could be clearing this up. We're still not sure what was compromised and how much. I would like to take this to the authorities.
